File: src/main/java/pages/CartPage.java
package pages;

import org.openqa.selenium.WebElement;
import org.openqa.selenium.support.FindBy;
import org.openqa.selenium.support.PageFactory;

public class CartPage extends BasePage {
    @FindBy(xpath = "//tr[@class='success']/td[2]")
    WebElement titleFirstModelAdd;
    @FindBy(xpath = "//tr[@class='success']/td[3]")
    WebElement priceFirstModelAdd;
    @FindBy(xpath = "//button[text()='Place Order']")
    WebElement buttonPlaceOrder;

    public CartPage() {
        this.driver = getDriver();
        PageFactory.initElements(driver, this);
    }

    public String getModelCart() {
        return getTextFromWebElement(titleFirstModelAdd);
    }

    public String getPriceCart() {
        return getTextFromWebElement(priceFirstModelAdd);
    }

    public void clickButtonPlaceOrder() {
        clickOnWebElement(buttonPlaceOrder);
    }
}

File: Desktop/proyectoWdio/test/pageobjects/cartPage.js
const Page = require('./page');

/**
 * sub page containing specific selectors and methods for a specific page
 */
class CartPage extends Page {
    /**
     * define selectors using getter methods
     */
    get titleFirstModelAdd(){
        return $("//tr[@class='success']/td[2]")
    };
    get priceFirstModelAdd(){
        return $("//tr[@class='success']/td[3]")
    };
    get buttonPlaceOrder(){
        return $("//button[text()='Place Order']")
    };

    async getModelCart() {
        await this.getTextFromWebElement(this.titleFirstModelAdd);
    }
    async getPrice() {
        await this.getPriceCart(this.priceFirstModelAdd);
    }

    async getclickButtonPlaceOrder() {
        await this.clickOnWebElement(this.buttonPlaceOrder);
    }
}

module.exports = new CartPage();